### PR TITLE
feat: boost tab with boosts count

### PR DESCRIPTION
**Pull request by ChaituVR**
### Summary

Closes: https://github.com/snapshot-labs/workflow/issues/323

- Add a new tab boost with count of boosts
image

### How to test

1. Go to http://127.0.0.1:8080/#/s:acrossprotocol.eth/proposal/0x078c419b439cf5e2889c3e2a14ac2953d8b3b033cf313254b61be4fbbc333483
2. You should see a new tab "Boost" with the count 
3. On click, it should redirect to v1

### CODE REVIEW

File: apps/ui/src/views/Proposal.vue [hunk 1]
```diff
@@ -116,7 +118,17 @@ watch(
...
 
     if (discussion.value) {
-      discourseTopic.value = await loadSingleTopic(discussion.value);
+      try {
+        discourseTopic.value = await loadSingleTopic(discussion.value);
+      } catch (error) {
+        console.error(error);
+      }
+    }
+    if (props.space.additionalRawData?.boost?.enabled) {
```
Comment (bonustrack) [inline apps/ui/src/views/Proposal.vue:127]:
```suggestion
    }
    
    if (props.space.additionalRawData?.boost?.enabled) {
```

**Comment by bonustrack** [comment]
Some comments:
- v1 UI should open in a new tab with `_blank` target
- We should not wait for discussion to finish loading to load boost count

File: apps/ui/src/views/Proposal.vue [hunk 2]
```diff
@@ -116,7 +118,18 @@ watch(
...
     if (discussion.value) {
-      discourseTopic.value = await loadSingleTopic(discussion.value);
+      try {
+        discourseTopic.value = await loadSingleTopic(discussion.value);
+      } catch (error) {
+        console.error(error);
+      }
+    }
+    
+    if (props.space.additionalRawData?.boost?.enabled) {
```
Comment (wa0x6e) [inline apps/ui/src/views/Proposal.vue:128]: This global setting will hide previous proposals with boost, created when the space has enabled boost (but not currently)

Comment (wa0x6e) [reply apps/ui/src/views/Proposal.vue:128]: Ideally, boost setting should also be saved on proposal, like `privacy`

Comment (ChaituVR) [reply apps/ui/src/views/Proposal.vue:126]: Hmm right now it is enabled globally or disabled globally at space level on v1

Comment (bonustrack) [reply apps/ui/src/views/Proposal.vue:126]: > This global setting will hide previous proposals with boost, created when the space has enabled boost (but not currently)

That's the expected behavior

**Review by bonustrack — APPROVED** [review]
tACK

**Pull request merged**